frontend/src/theming/index.less

```diff
@@ -1,4 +1,13 @@
 @import '~antd/dist/antd.dark.less';
+@import 'antd-button-color/dist/css/style.less';
+
+// global vars in less
```

Comment:
Please remove commented code, if this is needed for something else add a comment that motivates it

Reply:
Changed applied [here](https://github.com/netgroup-polito/CrownLabs/pull/460/commits/77c8d7a732e466080c9e305ba922259e2ecdaa42)